hack/jenkins/e2e-runner.sh

```diff
@@ -133,9 +154,19 @@ function dump_cluster_logs() {
     fi
 }
 
+# Have cmd/e2e run by goe2e.sh generate JUnit report in ${WORKSPACE}/junit*.xml
+ARTIFACTS=${WORKSPACE}/_artifacts
+mkdir -p ${ARTIFACTS}
+
+# E2E runner stages
```

Comment:
I think I might prefer not using these variables and just saying `record_command "CLEANUP" ...` or something. not a big deal.